Packs/Infoblox/Integrations/Infoblox/Infoblox.yml

```diff
@@ -791,6 +803,231 @@ script:
+      type: string
+    - contextPath: Infoblox.ModifiedResponsePolicyZoneRules.Canonical
+      description: Canonical value of the rule.
+      type: string
+    - contextPath: Infoblox.ModifiedResponsePolicyZoneRules.Disable
+      description: Whether the rule is disabled.
+      type: boolean
+    - contextPath: Infoblox.ModifiedResponsePolicyZoneRules.Comment
+      description: The comment for this rule.
+      type: string
+    - contextPath: Infoblox.ModifiedResponsePolicyZoneRules.Type
+      description: The object type as used in Infoblox.
+      type: string
+    - contextPath: Infoblox.ModifiedResponsePolicyZoneRules.View
+      description: View of the definition.
+      type: string
+    - contextPath: Infoblox.ModifiedResponsePolicyZoneRules.Zone
+      description: The zone to which this rule belongs.
+      type: string
+    - contextPath: Infoblox.ModifiedResponsePolicyZoneRules.ReferenceID
+      description: The reference ID of the rule.
+      type: string
+  - name: infoblox-create-host-record
+    description: "Creates a host record. Note: Either ipv4_address or ipv6_address is required."
+    arguments:
+    - name: name
+      description: The name of the host in FQDN format '{name}.{zone}' (e.g. example.auth_zone). It must belong to an existing DNS zone.
+      required: true
+    - name: view
+      description: The DNS view in which the records are located.
+      defaultValue: default
+      required: false
+    - name: ipv4_address
+      type: textArea
+      description: "The IPv4 addresses for the host record. Example: [{\"ipv4addr\": \"192.168.1.0\"}], [{\"ipv4addr\": \"192.168.1.1\", \"configure_for_dhcp\": true, \"mac\": \"00:01:00:01:00:01\"}]."
+      required: false
+    - name: ipv6_address
+      type: textArea
+      description: "The IPv6 addresses for the host record. Example: [{\"ipv6addr\": \"2001:db8::1\"}], [{\"ipv6addr\": \"2001:db8::2\", \"configure_for_dhcp\": true, \"duid\": \"00:01:00:01:0a:1a\"}]."
+      required: false
+    - name: comment
+      description: The comment for this host record.
+      required: false
+    - name: aliases
+      type: textArea
+      description: "The aliases for this host record. Example: [\"alias1\", \"alias2\"]."
+      required: false
+    - name: configure_for_dns
+      description: Whether to configure the host record for DNS.
+      required: false
+      defaultValue: true
+    - name: extattrs
+      type: textArea
+      description: "The extended attributes for this host record. Example: {\"Site\": {\"value\": \"example\"}}."
+      required: false
+    - name: additional_parameters
+      auto: PREDEFINED
+      type: keyValue
+      description: "JSON object containing additional parameters to create host record. Example: {\"use_ttl\": true, \"ttl\": 3600, \"disable\": true}."
+      required: false
+    outputs:
+    - contextPath: Infoblox.Host.ReferenceID
+      description: The host record reference ID.
+      type: String
```

Comment:
In the outputs for the `infoblox-create-host-record` command, several `type` values are capitalized (e.g., `String`, `Boolean`). Per XSOAR standards, these should be in lowercase (`string`, `boolean`). Please correct the types for all outputs in this command from line 908 to 947.